scripts/pwa-kit-deps-version.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Nit: Can we rename this file to `bump-pwa-kit-deps-version`? It is longer but give you a bit of a context what it does